lib/auth/password_test.go

```diff
@@ -126,13 +129,22 @@ func (s *PasswordSuite) TestTiming(c *C) {
 }
 
 func (s *PasswordSuite) TestChangePassword(c *C) {
+	s.a.IAuditLog = s.mockedAuditLog
+
 	req, err := s.prepareForPasswordChange("user1", []byte("abc123"), teleport.OFF)
 	c.Assert(err, IsNil)
 
 	fakeClock := clockwork.NewFakeClock()
 	s.a.SetClock(fakeClock)
 	req.NewPassword = []byte("abce456")
 
+	// test change password event
+	s.mockedAuditLog.MockEmitAuditEvent = func(event events.Event, fields events.EventFields) error {
+		c.Assert(event, DeepEquals, events.UserPasswordChange)
+		c.Assert(fields[events.EventUser], Equals, "user1")
+		return nil
+	}
```

Comment:
also check that `EmitAuditEvent` was actually called

Reply:
revised